app/services/nodes/merger.rb

```diff
@@ -90,15 +92,33 @@ def update_properties
   end
 
   def copy_attachments
```

Comment:
This class is including the `AttachmentsCopier` that defines a `copy_attachments` method but this also defines a method with the same name. Should we differentiate the 2 and use a different name?